packages/one-trust-notices/src/config/config.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  AWSConfig,
+  FileManagerConfig,
+  LoggerConfig,
+  S3Config,
+} from "pagopa-interop-commons";
+import { z } from "zod";
+import dotenv from "dotenv";
+
+export const OnetrustServiceConfig = FileManagerConfig.and(S3Config)
+  .and(LoggerConfig)
+  .and(AWSConfig)
+  .and(
+    z
+      .object({
+        LANGS: z.string().transform((value) => value.split(",")),
+        CONTENT_STORAGE_BUCKET: z.string(),
+        HISTORY_STORAGE_BUCKET: z.string(),
+        ONETRUST_CLIENT_ID: z.string(),
+        ONETRUST_CLIENT_SECRET: z.string(),
+        PRIVACY_NOTICES_UPDATER_TERMS_OF_SERVICE_UUID: z.string(),
+        PRIVACY_NOTICES_UPDATER_PRIVACY_POLICY_UUID: z.string(),
+        PRIVACY_NOTICES_DYNAMO_TABLE_NAME: z.string(),
+      })
+      .transform((c) => ({
+        langs: c.LANGS,
+        contentStorageBucket: c.CONTENT_STORAGE_BUCKET,
+        historyStorageBucket: c.HISTORY_STORAGE_BUCKET,
+        onetrustClientId: c.ONETRUST_CLIENT_ID,
+        onetrustClientSecret: c.ONETRUST_CLIENT_SECRET,
+        privacyNoticesUpdaterTermsOfServiceUuid:
+          c.PRIVACY_NOTICES_UPDATER_TERMS_OF_SERVICE_UUID,
+        privacyNoticesUpdaterPrivacyPolicyUuid:
+          c.PRIVACY_NOTICES_UPDATER_PRIVACY_POLICY_UUID,
+        privacyNoticesDynamoTableName: c.PRIVACY_NOTICES_DYNAMO_TABLE_NAME,
+      }))
+  );
+
+export type OnetrustServiceConfig = z.infer<typeof OnetrustServiceConfig>;
+
+dotenv.config();
```

Comment:
🧹  Unneeded? We don't call it in any other config file

Reply:
I do not know if it is needed it was present in the old script. Should we keep it?

Reply:
You can safely remove it, not needed with this way of managing configs :)